models/index.js

```diff
@@ -1,20 +1,17 @@
-const Sequelize = require('sequelize');
-const UserModel = require('./user');
-const CourseModel = require('./courses');
-const lessonsModel = require('./lessons');
-const CandidateModel = require('./candidate');
-const RecruiterModel = require('./recruiter');
-const QuestionModel = require('./question');
-const CandidateLessonsModel = require('./candidateLessons');
-const PersonaModel = require('./persona');
-const PermissionModel = require('./permissions');
-const TagModel = require('./tag');
-const LessonTagModel = require('./lessonTags');
-const PathModel = require('./path');
+const { sequelize, Sequelize } = require('../config/config');
 
-const sequelize = new Sequelize(process.env.DATABASE, process.env.DATABASE_USER, process.env.DATABASE_PASSWORD, {
-  dialect: 'postgres',
-});
+const UserModel = require('./authentication/user');
```

Comment:
I generally like this section, but it could be better organized by your sub-folders. Right now it's all jumbled.

e.g.
```
// authentication 
const UserModel = require('./authentication/user');
[ the rest of the stuff from the authentication folder ]

// learning-resources
const CourseModel = require('./learning-resources/courses);
[ the rest of the stuff from the learning-resources folder ]

etc.
```